caas/kubernetes/provider/k8s.go

```diff
@@ -933,7 +933,7 @@ func (k *kubernetesClient) Units(appName string) ([]caas.Unit, error) {
 			unitInfo.FilesystemInfo = append(unitInfo.FilesystemInfo, caas.FilesystemInfo{
 				StorageName:  storageName,
 				Size:         uint64(vol.PersistentVolumeClaim.Size()),
-				FilesystemId: string(pvc.UID),
+				FilesystemId: pvc.Name,
```

Comment:
not sure if the change from `UID` to `Name` impacts other places that use `FilesystemId` or not?

Reply:
This was a driveby to improve the id displayed to the user - it's informational only.

Reply:
ic. yes, Name makes more sense for human.